assets/base/i18n/ko.json

```diff
@@ -10,7 +10,7 @@
   "about.teamEditiont0": "팀 에디션",
   "about.teamEditiont1": "엔터프라이즈 에디션",
   "about.title": "{appTitle} 정보",
-  "announcment_banner.dont_show_again": "Don't show again",
+  "announcment_banner.dont_show_again": "",
```

Comment:
@metanerd can this be a problem with Weblate? some french strings are empty and korean has a bunch of them empty